dind_jenkins_pipeline.txt

```diff
@@ -0,0 +1,70 @@
+DInd: Docker inside Docker:
+1.	Check where the .pem file is located in the system and go that folder.
+2.	Copy the ssh link from the EC2 server and paste it.
+Ex: PS C:\Users\S.Tummalapalli\Downloads> ssh -i "amma.pem" ubuntu@ec2-18-193-76-75.eu-central-1.compute.amazonaws.com
+3.	Check for docker with the command: docker version (Otherwise follow the steps from the docs.docker.com to install respective to your OS)
+4.	Check for docker images if already Jenkins is running in it with the command: docker images.
+5.	Otherwise, this time will try with Dockerfile.
```

Comment:
I see this content above, if this the same info as readme.md, remove this text file